src/enums/index.ts

```diff
@@ -27,5 +27,6 @@ export {
   EffectiveDateTypes,
   EntityStates,
   FilingCodes,
-  StaffPaymentOptions
+  StaffPaymentOptions,
+  RelationshipTypes
```

Comment:
Please sort this list. It's not a big deal for a small list like this, but for larger lists it saves times/errors when looking for something.

Reply:
Noted.